src/android/patches/makeStringsPatch.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function makeStringsPatch(params) {
+  const patch = Object.keys(params).map(key =>
+    `    <string moduleConfig="true" name="${key}">${params[key]}</string>`
```

Comment:
Also we can define other, non string parameters (that should be easier with `xmldoc` as I proposed in previous comment), see http://developer.android.com/guide/topics/resources/more-resources.html

So `params` array could accept a type - this is mainly useful when you want to store a number instead of a String - not sure if there's any use-case for that, but good to know.